### ISSUE TITLE
Opening file and saving (without changes) shifts json order

### ISSUE THREAD

**Issue by miguelsolorio** [kactus-io/kactus#19]
### Description
Opening a file that's already been synced and simply saving (without making any changes) causes the files to change/modify, which creates a ripple on change on multiple files. It appears to only shift data around instead of adding/removing them:
![image](https://user-images.githubusercontent.com/1030624/28332811-e2ea2d60-6baa-11e7-9a09-7fde10f9268b.png)

### Version
**Kactus version:** 0.1.7

**OS version:** macOS Sierra 10.12.5 (16F73)

### Steps to Reproduce

1. Open Kactus
2. Pull any pending changes
3. Open Sketch file
4. Save file
5. Go back to Kactus and see pending changes

**Expected behavior:** No pending changes

**Actual behavior:** Pending changes

**Reproduces how often:** 100%

### Logs
```
2017-07-18T16:47:43.506Z - warn: [ui] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T16:47:43.761Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T16:53:30.432Z - info: [ui] Executing push: git -c credential.helper= push origin master:master --progress (took 4.401s)
2017-07-18T16:55:00.624Z - info: [ui] Executing push: git -c credential.helper= push origin master:master --progress (took 4.659s)
2017-07-18T17:44:55.169Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T17:55:32.968Z - error: [ui] `git -c credential.helper= pull --no-rebase --progress origin` exited with an unexpected code: 1.
Removing test_doc/images/4badf66d98f4969a7e1bae8aa699439ee71b07f9.png
Auto-merging test_doc/Symbols/9B6DEC8C-87ED-4781-8C06-68BB1AD80D83/symbolMaster.json
CONFLICT (content): Merge conflict in test_doc/Symbols/9B6DEC8C-87ED-4781-8C06-68BB1AD80D83/symbolMaster.json
Auto-merging test_doc/Symbols/9B6DEC8C-87ED-4781-8C06-68BB1AD80D83/10D3AB17-EF6A-4F83-8F2B-9CD4E8E5A7A0/shapeGroup.json
Auto-merging test_doc/Symbols/66BAAB3B-067D-495F-95E0-9EFF278A2CBC/EC5D34EC-20C5-401D-8C7B-25A5D012C704/shapeGroup.json
Auto-merging test_doc/Symbols/66BAAB3B-067D-495F-95E0-9EFF278A2CBC/D9C9267D-89E4-4DC6-B58A-B66C01CE9C00.json
Automatic merge failed; fix conflicts and then commit the result.

(The error was parsed as 7: We found some conflicts while trying to merge. Please resolve the conflicts and commit the changes.)
2017-07-18T18:02:46.654Z - info: [ui] Executing push: git -c credential.helper= push origin master:master --progress (took 4.622s)
2017-07-18T18:03:05.089Z - info: [ui] Executing push: git -c credential.helper= push origin master:master --progress (took 4.397s)
2017-07-18T18:06:39.217Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T18:06:39.257Z - info: [ui] Opt in reported.
2017-07-18T18:06:39.729Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T18:06:39.786Z - info: [ui] Stats reported.
2017-07-18T18:12:00.846Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T18:12:01.117Z - warn: [shared] checkUnlockedKactus: 'https://api.kactus.io/v1/1030624' returned a 404
2017-07-18T18:12:20.890Z - info: [ui] Executing push: git -c credential.helper= push origin master:master --progress (took 4.325s)
2017-07-18T18:22:50.248Z - info: [ui] Executing fetch: git -c credential.helper= fetch --progress --prune origin (took 1.709s)
```

#### Additional Information
Tested this with another user making changes at the same time. We previously resolved conflicts before seeing this issue.

**Comment by mathieudutour** [comment]
Very good point. I have a fix, will deployed in a bit

**Comment by mathieudutour** [comment]
fixed in 0.1.8 (deploying right now)
